SpatialGDK/Documentation/content/get-started/tutorial.md

```diff
@@ -65,6 +65,71 @@ In this section you’ll run a [local deployment](https://docs.improbable.io/ref
 3. Back in your two Unreal game clients, run around and shoot.
 4. Using the Inspector to track the location of your two players, notice that if you position them in the area of authority then their shots damage each other, but if they are on different servers, they can’t damage each other. Let’s fix that.
 
+### Replicate health changes
+
+In the ThirdPersonShooter project each `TPSCharacter` has a variable tracking health called `CurrentHealth`. The health deducted on the server when the character is being shot, however as the variable is not setup for replication it does not replicate to the client. Therefore the characters health meter does not update on the client even though the health variable is being depleted on the server. 
+
+To resolve this we need to mark the `CurrentHealth` property for replication as you would with normal [Unreal Actor replication](https://docs.unrealengine.com/en-us/Resources/ContentExamples/Networking/1_1):
+
+Navigate to the declaration of the `CurrentHealth` variable in the `TPSCharacter.h`(line 175) and change the line:
```

Comment:
Perhaps we shouldn't say "change the line" but rather "add the UProperty specifiers `ReplicatedUsing = OnRep_CurrentHealth`"

Reply:
We need to standardise how we present `go to line`.

Lets go with `TPSCharacter.h - line 175`

So overall this line should be 

Navigate to the declaration of the `CurrentHealth` variable in the `TPSCharacter.h` - line 175 and add the UProperty specifiers `ReplicatedUsing = OnRep_CurrentHealth`: